src/main/java/edu/kis/vh/nursery/DefaultCoutingOutRhymer.java

```diff
@@ -1,36 +1,34 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntArrayStack;
-import edu.kis.vh.nursery.list.IntLinkedList;
-
+import edu.kis.vh.nursery.Containers.*;
 public class DefaultCoutingOutRhymer {
-	private IntLinkedList intLinkedList;
+	private StackInterface stackInterface;

```

Comment:
"interface" suffix of the attribute name is wrong and does not show the meaning.